lib/datadog/grok/src/parse_grok.rs

```diff
@@ -0,0 +1,363 @@
+use itertools::{
+    FoldWhile::{Continue, Done},
+    Itertools,
+};
+
+use lookup::LookupBuf;
+use shared::btreemap;
+
+use crate::grok_filter::apply_filter;
+use crate::insert_field::insert_field;
+use crate::parse_grok_rules::GrokRule;
+use tracing::warn;
```

Comment:
Nit: In other libs, we're using `#[macro_use] extern crate tracing;` to pull `warn!` into scope at the crate level.